### PR TITLE
Guide for contributing 

### PR DESCRIPTION
**Pull request by Agheb**
Please provide feedback and check for typos. Need a review ASAP 
Thanks !

### CODE REVIEW

**Comment by Agheb** [comment]
@euchenhofer  What's your take on this ?

**Comment by jvaes** [comment]
Nice work @Agheb!

**Review by adomanskagrzyb — APPROVED** [review]

**Comment by adomanskagrzyb** [comment]
Very clean and easy to understand - great work!

File: contributing.md [hunk 1]
```diff
@@ -1,30 +1,77 @@
...
+- Fun and Interactive GitHub Challenge: [Got 15 minutes and want to learn Git?](https://try.github.io/levels/1/challenges/1)
+- [Git - The simple guide](http://rogerdudler.github.io/git-guide/)
+
+#### Further Reading
 
 - [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)
+- [How to Write a Git Commit Message - The seven rules for a great commit message](http://chris.beams.io/posts/git-commit/#seven-rules)
+- [Distributed Git - Contributing to a Project]( http://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines)
+- [Thoughtbot - 5 Useful Tips For A Better Commit Message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message)
+- [Tim Pope - A Note About Git Commit Messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
```
Comment (adomanskagrzyb) [inline contributing.md:81]: I would add something like `last but not least - if you are creating a pull request (and you are not on contributor's list) please add yourself`.

**Comment by Agheb** [comment]
Done @DomanskaGrzyb @jvaes

**Comment by Stefan0x** [comment]
@agheb Very cool! Good job :)

**Comment by Stefan0x** [comment]
While going through the Git Course on Udacity, I found this excellent **GIT CHEAT SHEET** (Officially created by GitHub): https://github.com/github/training-kit/blob/master/downloads/github-git-cheat-sheet.pdf

Maybe with that, we could even remove the "How-To" section like you suggested, Agheb?

**Comment by Agheb** [comment]
@euchenhofer I think we should keep the How-to section because the cheat sheet doesn't describe the workflow of how to submit a PR.

**Pull request merged**